src/lib/markbind/src/utils/nunjuckUtils.js

```diff
@@ -0,0 +1,31 @@
+const START_ESCAPE_STR = '{% raw %}';
+const END_ESCAPE_STR = '{% endraw %}';
+const REGEX = new RegExp('{% *raw *%}(.*?){% *endraw *%}', 'gs'); // eslint-disable-line no-useless-escape
+
+function addEscapeTags(match) {
+  return `${START_ESCAPE_STR}${match}${END_ESCAPE_STR}`;
+}
+
+function removeEscapeTags(_, p1) {
+  return p1;
+}
+
+function preEscapeRawTags(pageData) {
+  return pageData.replace(REGEX, addEscapeTags);
+}
+
+module.exports = {
+  renderEscaped(nunjucks, pageData, variableMap = {}, options = {}) {
+    const escapedPage = preEscapeRawTags(pageData);
+    return nunjucks.renderString(escapedPage, variableMap, options);
+  },
+
+  /**
+   * RemoveNunjucksEscapes removes raw tags from page data when processing
+   * Downstream calls to nunjuckUtils.renderEscaped will be essentially the same as nunjucks.renderString
+   * This ensures that we remove {{ and }} from the final output which prevents problem with Vue
+   */
+  removeNunjucksEscapes(pageData) {
+    return pageData.replace(REGEX, removeEscapeTags);
```

Comment:
Just realised there's the `$&` replacement pattern; Could help make this even more succinct 😄 
I'm fine with using a function as well though. ( likewise for `preEscapeRawTags` )

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter